src/stylesheets/components/_process-list.scss

```diff
@@ -0,0 +1,65 @@
+// PROCESS LIST
+$theme-process-list-bullet-border-width: units($theme-process-list-connection-line-width) / 2;
+$theme-process-list-bullet-content-padding: (units($theme-process-list-bullet-size) / 2) - units(0.5);
+$theme-process-list-bullet-line-x-position: (
+  (units($theme-process-list-bullet-size) / 2) - (units($theme-process-list-connection-line-width) / 2)
+);
+
+.usa-process-list {
+  @include typeset($theme-process-list-font-family);
+  @include unstyled-list;
+  counter-reset: usa-numbered-list;
+  padding: 20px 0 0 $theme-process-list-bullet-line-x-position;
+  position: relative;
+}
+
+.usa-process-list__item {
+  border-left: units($theme-process-list-connection-line-width) solid color($theme-process-list-connection-line-color);
+  font-size: size($theme-process-list-font-family, $theme-process-list-font-size);
+  padding-bottom: units(4);
+  padding-left: calc(#{$theme-process-list-bullet-content-padding} * 2);
+  &:last-child {
+    border-left: units($theme-process-list-connection-line-width) solid transparent;
+  }
+  &::before {
+    @include u-flex("row", "align-center", "justify-center");
+    $half-of-font-size: -0.4rem;
+    background-color: color($theme-process-list-bullet-background-color);
+    border-radius: 99rem;
```

Comment:
Might be a token for making circles.

Reply:
There's a mixin u-circle() which is used in step-indicator. I changed it to that.